packages/drivers/odsp-driver/src/odspDocumentServiceFactoryWithCodeSplit.ts

```diff
@@ -24,7 +24,7 @@ export class OdspDocumentServiceFactoryWithCodeSplit
         super(
             getStorageToken,
             getWebsocketToken,
-            async () => import("./getSocketIo").then((m) => m.getSocketIo()),
+            async () => import(/* webpackChunkName: "socketModule" */ "./getSocketIo").then((m) => m.getSocketIo()),
```

Comment:
It's not clear we need this chunk, and OdspDocumentServiceFactoryWithCodeSplit overall, if bigger part of socket-related code is in separate bundle. I think we need to deprecate it and have only one factory